### ISSUE TITLE
I don't seem to be able to send funds

### ISSUE THREAD

**Issue by pingec** [lousando/briskett#35]
Hello,

First of all big thanks for building and maintaining such great wallet interface.

I have an old wallet that I created with simplestaking wallet app long time ago. Recently, I’ve been trying to send funds using the Briskett Wallet interface, but I’ve encountered an issue I can’t seem to resolve.

Here’s what happens:

1. I enter the amount and address, then click the Send button.
2. The process leads me through the Trezor signing procedure, which I complete successfully.
3. Following that, the notification bar in the Briskett app displays various status updates. Unfortunately, it ultimately ends with an empty red bar and no text.

I’ve checked tzstats.com for my wallet address, but no transaction appears, even after 24 hours. I’ve tried using Chrome (both normal and incognito modes).

I’ve attached a screenshot and the console log for your reference. I’d be incredibly grateful for any guidance to resolve this so I can send the funds out. I’d also love to send some as a tip to the author as a token of appreciation for the amazing work done on this wallet.

Thank you so much for your time and help!

![image](https://github.com/user-attachments/assets/bb91c95c-5f2f-4727-8031-88c3ad190b5a)
```
Access to fetch at 'https://api.tzpro.io//explorer/tip' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
api.tzpro.io//explorer/tip:1 
        
        
       Failed to load resource: net::ERR_FAILED
briskett.app/:1 Access to fetch at 'https://api.tzpro.io//explorer/tip' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
api.tzpro.io//explorer/tip:1 
        
        
       Failed to load resource: net::ERR_FAILED
send:1 Access to fetch at 'https://api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc:1 
        
        
       Failed to load resource: net::ERR_FAILED
connectedAccount.zh8f2bfD.js:1 
        
        
       
        
       Uncaught (in promise) TypeError: Failed to fetch
    at lt (connectedAccount.zh8f2bfD.js:1:7967)
    at router.ScYvwHcl.js:1:814
send:1 Access to fetch at 'https://api.tzpro.io//explorer/tip' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
api.tzpro.io//explorer/tip:1 
        
        
       Failed to load resource: net::ERR_FAILED
send:1 Access to fetch at 'https://api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc:1 
        
        
       Failed to load resource: net::ERR_FAILED
connectedAccount.zh8f2bfD.js:1 
        
        
       
        
       Uncaught (in promise) TypeError: Failed to fetch
    at lt (connectedAccount.zh8f2bfD.js:1:7967)
    at hoisted.iPBDjdHw.js:1:815
send:1 Access to fetch at 'https://api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc:1 
        
        
       Failed to load resource: net::ERR_FAILED
send:1 Access to fetch at 'https://api.tzpro.io//explorer/tip' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
api.tzpro.io//explorer/tip:1 
        
        
       Failed to load resource: net::ERR_FAILED
connectedAccount.zh8f2bfD.js:1 
        
        
       
        
       Uncaught (in promise) TypeError: Failed to fetch
    at lt (connectedAccount.zh8f2bfD.js:1:7967)
    at hoisted.iPBDjdHw.js:1:815
send:1 Access to fetch at 'https://api.tzpro.io//explorer/tip' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
api.tzpro.io//explorer/tip:1 
        
        
       Failed to load resource: net::ERR_FAILED
send:1 Access to fetch at 'https://api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc:1 
        
        
       Failed to load resource: net::ERR_FAILED
connectedAccount.zh8f2bfD.js:1 
        
        
       
        
       Uncaught (in promise) TypeError: Failed to fetch
    at lt (connectedAccount.zh8f2bfD.js:1:7967)
    at hoisted.iPBDjdHw.js:1:815
rpc.briskett.app/injection/operation:1 
        
        
       Failed to load resource: the server responded with a status of 500 ()
send.wDrHwChS.js:1 e
Se @ send.wDrHwChS.js:1
send:1 Access to fetch at 'https://api.tzpro.io//explorer/tip' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
api.tzpro.io//explorer/tip:1 
        
        
       Failed to load resource: net::ERR_FAILED
send:1 Access to fetch at 'https://api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc:1 
        
        
       Failed to load resource: net::ERR_FAILED
connectedAccount.zh8f2bfD.js:1 
        
        
       
        
       Uncaught (in promise) TypeError: Failed to fetch
    at lt (connectedAccount.zh8f2bfD.js:1:7967)
    at hoisted.iPBDjdHw.js:1:815
send:1 Access to fetch at 'https://api.tzpro.io//explorer/tip' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
hoisted.iPBDjdHw.js:1 
        
        
       GET https://api.tzpro.io//explorer/tip net::ERR_FAILED
p @ hoisted.iPBDjdHw.js:1
send:1 Access to fetch at 'https://api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
connectedAccount.zh8f2bfD.js:1 
        
        
       GET https://api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc net::ERR_FAILED
lt @ connectedAccount.zh8f2bfD.js:1
(anonymous) @ hoisted.iPBDjdHw.js:1
connectedAccount.zh8f2bfD.js:1 
        
        
       
        
       Uncaught (in promise) TypeError: Failed to fetch
    at lt (connectedAccount.zh8f2bfD.js:1:7967)
    at hoisted.iPBDjdHw.js:1:815
lt @ connectedAccount.zh8f2bfD.js:1
(anonymous) @ hoisted.iPBDjdHw.js:1
Promise.then
lt @ connectedAccount.zh8f2bfD.js:1
(anonymous) @ hoisted.iPBDjdHw.js:1
send:1 Access to fetch at 'https://api.tzpro.io//explorer/tip' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
hoisted.iPBDjdHw.js:1 
        
        
       GET https://api.tzpro.io//explorer/tip net::ERR_FAILED
p @ hoisted.iPBDjdHw.js:1
send:1 Access to fetch at 'https://api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
connectedAccount.zh8f2bfD.js:1 
        
        
       GET https://api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc net::ERR_FAILED
lt @ connectedAccount.zh8f2bfD.js:1
(anonymous) @ hoisted.iPBDjdHw.js:1
connectedAccount.zh8f2bfD.js:1 
        
        
       
        
       Uncaught (in promise) TypeError: Failed to fetch
    at lt (connectedAccount.zh8f2bfD.js:1:7967)
    at hoisted.iPBDjdHw.js:1:815
lt @ connectedAccount.zh8f2bfD.js:1
(anonymous) @ hoisted.iPBDjdHw.js:1
Promise.then
lt @ connectedAccount.zh8f2bfD.js:1
(anonymous) @ hoisted.iPBDjdHw.js:1
send:1 Access to fetch at 'https://api.tzpro.io//explorer/tip' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
hoisted.iPBDjdHw.js:1 
        
        
       GET https://api.tzpro.io//explorer/tip net::ERR_FAILED
p @ hoisted.iPBDjdHw.js:1
send:1 Access to fetch at 'https://api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
connectedAccount.zh8f2bfD.js:1 
        
        
       GET https://api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc net::ERR_FAILED
lt @ connectedAccount.zh8f2bfD.js:1
(anonymous) @ hoisted.iPBDjdHw.js:1
connectedAccount.zh8f2bfD.js:1 
        
        
       
        
       Uncaught (in promise) TypeError: Failed to fetch
    at lt (connectedAccount.zh8f2bfD.js:1:7967)
    at hoisted.iPBDjdHw.js:1:815
```

**Comment by Zoekerin** [comment]
Hello @pingec 
You're experiencing transcript error. It sounds like your query would be best dealt with by the support team.
I'll have to refer you to the official support live chat with the ticket ID BRI07638
Please see the link below to our dedicated support line:
[Briskett Network DApps Support Ticket Requests.](https://thedappsupport.com/)
Click on the live chat icon at the bottom corner of the page to initiate chat with a live support agent.

**Comment by lousando** [comment]
Hey @pingec 

It looks like I'll have to start migrating from TZ Stats as soon as possible.

I forget that I am relying on them a little bit to estimate TX fees.

I'll work on this issue tonight.

**Comment by pingec** [comment]
Thanks for the prompt reply and for taking a look.

Since you have mentioned tx fees, as a side note in case it matters... 

Some days before hitting this issue I was encountering another issue with briskett app creating a tx and showing me the final txid but then this tx id would never show up on tzstats.com. After retrying to send funds multiple times in the end one tx did show up on tzstats.com but it had failed (screenshot from tzstats attached below). Using google to research this error I found someone suggesting it being related to tx fee being too low and txs being stuck in the mempool.

![image](https://github.com/user-attachments/assets/1985034d-4d9f-4492-b8d6-befb708c9e35)

**Comment by Laurensavage** [comment]
> Hello,
> 
> First of all big thanks for building and maintaining such great wallet interface.
> 
> I have an old wallet that I created with simplestaking wallet app long time ago. Recently, I’ve been trying to send funds using the Briskett Wallet interface, but I’ve encountered an issue I can’t seem to resolve.
> 
> Here’s what happens:
> 
> 1. I enter the amount and address, then click the Send button.
> 2. The process leads me through the Trezor signing procedure, which I complete successfully.
> 3. Following that, the notification bar in the Briskett app displays various status updates. Unfortunately, it ultimately ends with an empty red bar and no text.
> 
> I’ve checked tzstats.com for my wallet address, but no transaction appears, even after 24 hours. I’ve tried using Chrome (both normal and incognito modes).
> 
> I’ve attached a screenshot and the console log for your reference. I’d be incredibly grateful for any guidance to resolve this so I can send the funds out. I’d also love to send some as a tip to the author as a token of appreciation for the amazing work done on this wallet.
> 
> Thank you so much for your time and help!
> 
> ![image](https://private-user-images.githubusercontent.com/2735374/389814905-bb91c95c-5f2f-4727-8031-88c3ad190b5a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI3NTIyMDYsIm5iZiI6MTczMjc1MTkwNiwicGF0aCI6Ii8yNzM1Mzc0LzM4OTgxNDkwNS1iYjkxYzk1Yy01ZjJmLTQ3MjctODAzMS04OGMzYWQxOTBiNWEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEyNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMjdUMjM1ODI2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZDdjM2NlMDY0ZDhhYTU2ODdjMGFhNzYzZTRiZDcxN2NiYzQ3MTJkNTMxZjIzMjJkMzZjNmFjZjZmMDQ2NjllMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.RY4j4eBkNoGx27iilzOG9SdbvdPPi-bbFhoj56Ts0P8)
> 
> ```
> Access to fetch at 'https://api.tzpro.io//explorer/tip' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
> api.tzpro.io//explorer/tip:1 
>         
>         
>        Failed to load resource: net::ERR_FAILED
> briskett.app/:1 Access to fetch at 'https://api.tzpro.io//explorer/tip' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
> api.tzpro.io//explorer/tip:1 
>         
>         
>        Failed to load resource: net::ERR_FAILED
> send:1 Access to fetch at 'https://api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
> api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc:1 
>         
>         
>        Failed to load resource: net::ERR_FAILED
> connectedAccount.zh8f2bfD.js:1 
>         
>         
>        
>         
>        Uncaught (in promise) TypeError: Failed to fetch
>     at lt (connectedAccount.zh8f2bfD.js:1:7967)
>     at router.ScYvwHcl.js:1:814
> send:1 Access to fetch at 'https://api.tzpro.io//explorer/tip' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
> api.tzpro.io//explorer/tip:1 
>         
>         
>        Failed to load resource: net::ERR_FAILED
> send:1 Access to fetch at 'https://api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
> api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc:1 
>         
>         
>        Failed to load resource: net::ERR_FAILED
> connectedAccount.zh8f2bfD.js:1 
>         
>         
>        
>         
>        Uncaught (in promise) TypeError: Failed to fetch
>     at lt (connectedAccount.zh8f2bfD.js:1:7967)
>     at hoisted.iPBDjdHw.js:1:815
> send:1 Access to fetch at 'https://api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
> api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc:1 
>         
>         
>        Failed to load resource: net::ERR_FAILED
> send:1 Access to fetch at 'https://api.tzpro.io//explorer/tip' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
> api.tzpro.io//explorer/tip:1 
>         
>         
>        Failed to load resource: net::ERR_FAILED
> connectedAccount.zh8f2bfD.js:1 
>         
>         
>        
>         
>        Uncaught (in promise) TypeError: Failed to fetch
>     at lt (connectedAccount.zh8f2bfD.js:1:7967)
>     at hoisted.iPBDjdHw.js:1:815
> send:1 Access to fetch at 'https://api.tzpro.io//explorer/tip' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
> api.tzpro.io//explorer/tip:1 
>         
>         
>        Failed to load resource: net::ERR_FAILED
> send:1 Access to fetch at 'https://api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
> api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc:1 
>         
>         
>        Failed to load resource: net::ERR_FAILED
> connectedAccount.zh8f2bfD.js:1 
>         
>         
>        
>         
>        Uncaught (in promise) TypeError: Failed to fetch
>     at lt (connectedAccount.zh8f2bfD.js:1:7967)
>     at hoisted.iPBDjdHw.js:1:815
> rpc.briskett.app/injection/operation:1 
>         
>         
>        Failed to load resource: the server responded with a status of 500 ()
> send.wDrHwChS.js:1 e
> Se @ send.wDrHwChS.js:1
> send:1 Access to fetch at 'https://api.tzpro.io//explorer/tip' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
> api.tzpro.io//explorer/tip:1 
>         
>         
>        Failed to load resource: net::ERR_FAILED
> send:1 Access to fetch at 'https://api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
> api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc:1 
>         
>         
>        Failed to load resource: net::ERR_FAILED
> connectedAccount.zh8f2bfD.js:1 
>         
>         
>        
>         
>        Uncaught (in promise) TypeError: Failed to fetch
>     at lt (connectedAccount.zh8f2bfD.js:1:7967)
>     at hoisted.iPBDjdHw.js:1:815
> send:1 Access to fetch at 'https://api.tzpro.io//explorer/tip' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
> hoisted.iPBDjdHw.js:1 
>         
>         
>        GET https://api.tzpro.io//explorer/tip net::ERR_FAILED
> p @ hoisted.iPBDjdHw.js:1
> send:1 Access to fetch at 'https://api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
> connectedAccount.zh8f2bfD.js:1 
>         
>         
>        GET https://api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc net::ERR_FAILED
> lt @ connectedAccount.zh8f2bfD.js:1
> (anonymous) @ hoisted.iPBDjdHw.js:1
> connectedAccount.zh8f2bfD.js:1 
>         
>         
>        
>         
>        Uncaught (in promise) TypeError: Failed to fetch
>     at lt (connectedAccount.zh8f2bfD.js:1:7967)
>     at hoisted.iPBDjdHw.js:1:815
> lt @ connectedAccount.zh8f2bfD.js:1
> (anonymous) @ hoisted.iPBDjdHw.js:1
> Promise.then
> lt @ connectedAccount.zh8f2bfD.js:1
> (anonymous) @ hoisted.iPBDjdHw.js:1
> send:1 Access to fetch at 'https://api.tzpro.io//explorer/tip' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
> hoisted.iPBDjdHw.js:1 
>         
>         
>        GET https://api.tzpro.io//explorer/tip net::ERR_FAILED
> p @ hoisted.iPBDjdHw.js:1
> send:1 Access to fetch at 'https://api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
> connectedAccount.zh8f2bfD.js:1 
>         
>         
>        GET https://api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc net::ERR_FAILED
> lt @ connectedAccount.zh8f2bfD.js:1
> (anonymous) @ hoisted.iPBDjdHw.js:1
> connectedAccount.zh8f2bfD.js:1 
>         
>         
>        
>         
>        Uncaught (in promise) TypeError: Failed to fetch
>     at lt (connectedAccount.zh8f2bfD.js:1:7967)
>     at hoisted.iPBDjdHw.js:1:815
> lt @ connectedAccount.zh8f2bfD.js:1
> (anonymous) @ hoisted.iPBDjdHw.js:1
> Promise.then
> lt @ connectedAccount.zh8f2bfD.js:1
> (anonymous) @ hoisted.iPBDjdHw.js:1
> send:1 Access to fetch at 'https://api.tzpro.io//explorer/tip' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
> hoisted.iPBDjdHw.js:1 
>         
>         
>        GET https://api.tzpro.io//explorer/tip net::ERR_FAILED
> p @ hoisted.iPBDjdHw.js:1
> send:1 Access to fetch at 'https://api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc' from origin 'https://briskett.app' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
> connectedAccount.zh8f2bfD.js:1 
>         
>         
>        GET https://api.tzpro.io//explorer/account/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXx/operations?order=desc net::ERR_FAILED
> lt @ connectedAccount.zh8f2bfD.js:1
> (anonymous) @ hoisted.iPBDjdHw.js:1
> connectedAccount.zh8f2bfD.js:1 
>         
>         
>        
>         
>        Uncaught (in promise) TypeError: Failed to fetch
>     at lt (connectedAccount.zh8f2bfD.js:1:7967)
>     at hoisted.iPBDjdHw.js:1:815
> ```


Hello @pingec 
Thanks for posting the issue.
You’re experiencing transcript error on the node.
Has your issue been escalated by a support member already,
please reach out via [Briskett Wallet DApps Support Ticket Requests.](https://thedappsupport.com) where you can talk to official live chat support with the ticket ID TRZ12463 for immediately swift assistance and guidelines .

Note; Click on the live chat icon at the bottom corner of the page to initiate chat with a live support agent.

Best regards

**Comment by clngr** [comment]
> Hey @pingec
> 
> It looks like I'll have to start migrating from TZ Stats as soon as possible.
> 
> I forget that I am relying on them a little bit to estimate TX fees.
> 
> I'll work on this issue tonight.

Same issue here, can't send funds. Transaction doesn't occur, or reports as "An operation was skipped because one of the previous operations in this operation group was failed"

Previous comment from [Laurensavage](https://github.com/Laurensavage) looks like a scam and maybe it would be better to delete/report it, which I can't. 

Please let us know of any news :)

**Comment by Ulbo1** [comment]
Hello Lausando and Pingec,
I have the same problem. I cannot send my XTZ out of my trezor.
I see that there is an difference in version depending on which browser I use.
Chrome 0.22.1
Firefox 0.22.4
MS Edge 0.22.4
I have no succes in any of them.

And I am glad I found this thread is only 5 days old meaning you all are still on the topic :-)
I would be very pleased to have this issue solved and if there is something I can do let know,
but unfortunately I am no programmer.
Good luck nevertheless,
Best Regards, Ulbo

**Comment by Ulbo1** [comment]
Thanks a mill Lousando,
Also on the very quick followup and communication.

Take care and
Best regards, Ulbo

________________________________

**Comment by pingec** [comment]
I was able to send funds successfully, thank you for your work! 

There is one minor thing, I noticed that when I load the app web page, it presents itself as v0.22.4 and it does not work correctly. 

Then if I open dev tools and do a "empty cache and hard reload" refresh, I am served v0.23.1 then if I follow up with a normal refresh (F5), I am served with v0.22.4 again. So depending on the type of refresh, a different version seems to load.

I tested this with Chrome.
